src/functionalTest/java/uk/gov/hmcts/reform/amapi/functional/AccessManangementApiTest.java

```diff
@@ -29,6 +29,7 @@
 
     Response response;
 
+    public static final String VERSION = "v1";
```

Comment:
instead of defining version as static variable in the class, can we read this from application-functional.yaml?

Reply:
updated. We can used from yaml for Function and controller and Integration tests. For unit and Gatling we can't use.